AmplifyPlugins/Logging/Sources/AWSCloudWatchLoggingPlugin/Persistence/LogFile.swift

```diff
@@ -48,32 +48,32 @@ final class LogFile {
             return 0
         }
     }
-    
+
     /// Attempts to close the underlying log file.
     func close() throws {
         try self.handle.close()
     }
-    
+
     /// Atempts to flush the receivers contents to disk.
     func synchronize() throws {
         try self.handle.synchronize()
     }
-    
+
     /// - Returns: true if writing to the underlying log file will keep its size below the limit.
     func hasSpace(for data: Data) -> Bool {
         return UInt64(data.count) <= self.available
     }
-    
+
     /// Writes the given **single line of text** represented as a
     /// Data  to the underlying log file.
-    func write(data: Data) throws {        
+    func write(data: Data) throws {
         if #available(macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4, *) {
             try self.handle.write(contentsOf: data)
         } else {
             self.handle.write(data)
         }
         try self.handle.synchronize()
-        count = count + UInt64(data.count)
+        count = count + UInt64(data.count) // swiftlint:disable:this shorthand_operator
```

Comment:
is this something to do with `+=` operator?

Reply:
yes, shorthand is preferred but I left it as is for readability